packages/account/src/Components/poi/status/unsupported/card-details/uploader.tsx

```diff
@@ -159,7 +159,7 @@ const Uploader = ({ data, value, is_full, onChange, has_frame }: Partial<TUpload
                                     value={image}
                                     has_frame={has_frame}
                                     setFieldValue={setFieldValue}
-                                    handleChange={() => handleChange()}
+                                    handleChange={handleChange}
```

Comment:
on line:24, `setFieldValue` is optional, there should be an eslint error on this line. Can you please verify?